app/main/views/signup.py

```diff
@@ -225,6 +225,14 @@ def render_application(id, step=None, substep=None):
         'user_email': current_user.email_address
     }
 
+    # Add service pricing
```

Comment:
please investigate using data_api_client.get_application(id) on line 211 and combine the data in that 1 request